admin/src/linter.rs

```diff
@@ -34,15 +34,15 @@ impl Linter {
     /// Create a new linter for the database at the given path
     pub fn new(repo_path: impl Into<PathBuf>) -> Result<Self, Error> {
         let repo_path = repo_path.into();
-        let http_client = ureq::AgentBuilder::new()
-            .user_agent("RustSec advisory database linter")
-            .timeout(Duration::from_secs(20))
-            .build();
+        let crates_index = crates_index::Index::new_cargo_default();
+        if !crates_index.exists() {
+            crates_index.retrieve()?;
```

Comment:
Is there a way to update it in case it exists, but it's stale?

Reply:
Good catch! Turns out the crate examples are misleading. Fixed!